src/diffusers/models/attention.py

```diff
@@ -577,17 +607,20 @@ def forward(self, hidden_states, context=None, mask=None):
         key = self.reshape_heads_to_batch_dim(key)
         value = self.reshape_heads_to_batch_dim(value)
 
-        # TODO(PVP) - mask is currently never used. Remember to re-implement when used
+        if mask is not None:
```

Comment:
let's rename it to `attention_mask` here as well

Reply:
done here! https://github.com/huggingface/diffusers/pull/1428/commits/2b30fbc7d384335f7aef047695b502dddb01cf16